.github/ISSUE_TEMPLATE/Bug_Report_PSScriptAnalyzer.md

```diff
@@ -0,0 +1,8 @@
+---
+name: PSScriptAnalyzer bug report 🚦
+about: PSScriptAnalyzer bugs are tracked in a separate GitHub repo.
+
+---
+
+* Please submit PSScriptAnalyzer issues to the
+  [PSScriptAnalyzer](https://github.com/powershell/PSScriptAnalyzer/issues) repo on GitHub.
```

Comment:
Do you know if hyperlink's in the `about:` field would work?  As a convenience it would be nice to click there without having the extra `Get Started` button click.

Reply:
No, they don't work, I tried in the test repo [here](https://github.com/bergmeister/GitHub_MultipleIssuesTemplates/issues/new/choose) once with a plain link and once with a markdown link.
It seems GitHub itself don't have their own website on GitHub...